.github/workflows/build.yml

```diff
@@ -32,6 +32,8 @@ jobs:
                 key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
                 restore-keys: ${{ runner.os }}-m2
           -   name: Build with Maven
+              run: mvn clean install
```

Comment:
What does it do?
I mean what's problem you're trying to solve with this change?

Reply:
The problem of building pr without access to secrets

Reply:
It's enough here `mvn clean verify`.
You shouldn't do install command in such cases. 

And the next step should be replaced with `mvn verify -P sonar` - without **clean**

Reply:
(it looks like a package is needed)